docs/basics.md

```diff
@@ -396,6 +396,39 @@ Here is an example of backends and servers definition:
 - `backend2` will forward the traffic to two servers: `http://172.17.0.4:80"` with weight `1` and `http://172.17.0.5:80` with weight `2` using `drr` load-balancing strategy.
 - a circuit breaker is added on `backend1` using the expression `NetworkErrorRatio() > 0.5`: watch error ratio over 10 second sliding window
 
+## Custom Error pages
+
+Custom error pages can be returned, in lieu of the default, according to frontend-configured ranges of HTTP Status codes.
+In the example below, if a 503 status is returned from the frontend "website", the custom error page at http://2.3.4.5/503.html is returned with the actual status code set in the HTTP header.
+Note, the 503.html page itself is not hosted on traefik, but some other infrastructure.   
+
+```toml
+[frontends]
+  [frontends.website]
+  backend = "website"
+  [errors]
```

Comment:
This bit of documentation looks wrong: the test fixture that resembles this has `[frontends.frontend1.errors]`
`[frontends.frontend1.errors.networks]`, but the doc here just says `[errors]` and `[errors.networks]`.

Reply:
Thanks for noticing! Could you please file a new issue or ideally submit a PR? This one has been closed a while ago.

Thank you.